src/controllers/bulk.js

```diff
@@ -5,7 +5,15 @@ class BulkController extends BaseController {
     super(kuzzle, 'bulk');
   }
 
-  import (data, options) {
+  /**
+   * Creates, updates or deletes large amounts of documents as fast as possible.
+   * https://docs.kuzzle.io/core/1/api/controllers/bulk/import/
```

Comment:
(nitpicking) use [@link](https://jsdoc.app/tags-inline-link.html):

```suggestion
   * {@link https://docs.kuzzle.io/core/1/api/controllers/bulk/import/|Official documentation}
```